app/view/menuitem/LayerMetadata.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * MenuItem to show a metadata window for a layer
+ *
+ * @class CpsiMapview.view.menuitem.LayerMetadata
+ */
+Ext.define('CpsiMapview.view.menuitem.LayerMetadata', {
+    extend: 'Ext.menu.Item',
+    xtype: 'cmv_menuitem_layermetadata',
+    requires: [
+        'CpsiMapview.util.Layer'
+    ],
+
+    /**
+     * The connected layer for this item.
+     *
+     * @cfg {ol.layer.Base}
+     */
+    layer: null,
+
+    /**
+     * Text shown in this MenuItem
+     * @cfg {String}
+     */
+    text: 'Metadata',
+
+    /**
+     * @private
+     */
+    initComponent: function () {
+        var me = this;
+
+        me.handler = me.handlerFunc;
+
+        me.callParent();
+    },
+
+    /**
+     * Executed when this menu item is clicked.
+     * Opens a window with metadata for the connected layer.
+     */
+    handlerFunc: function () {
+        var me = this;
+
+        /**
+         * Helper function to notify user
+         * that metadata is not availble
+         */
+        function alert_no_metadata(){
```

Comment:
Please move this outside of handlerFunc as member function. Also please use camel case notation

Reply:
fixed with 8e7108b5c8286adfce0795e7c372108b3748adf2